src/blocks/Mainframe/Pageframe.js

```diff
@@ -1,12 +1,36 @@
+import { useState, useEffect } from "react"
 import Head from "next/head"
+import { supabase } from "@/lib/supabaseClient";
+
 import { Navbar } from "./Navbar"
 import Footer from "./Footer"
+import { AppContext } from "@/contexts/AppContext";
 
 const siteDescription = "Supertype Collective is a community of analytics developers, data scientists &#38; engineering leaders building products across the full stack."
 
 const Pageframe = ({ children, title }) => {
+
+    const [isLoggedIn, setIsLoggedIn] = useState(false)
+
+    // check supabase to see if user is logged in
+    useEffect(() => {
+
+        async function checkUser() {
+            const { data, error } = await supabase.auth.getSession()
+
+            if (data.session) {
+                setIsLoggedIn(data ? {
+                    githubToken: data.session.access_token,
```

Comment:
I think we should change this to `data.session.provider_token`

Reply:
fixed! the latest commit mainly involves bringing in superinference for the /enroll flow